apps/src/code-studio/utils.js

```diff
@@ -48,3 +48,17 @@ export function updateQueryParam(param, value) {
   }
   window.history.pushState(null, document.title, newLocation);
 }
+
+/**
+ * We indicate that a page is Rtl by setting the dir attribute on the root element
```

Comment:
The reason we can't easily remove this from the DOM is that it's also used for a bunch of CSS :(